examples/app/templates/my-secret-ca.yaml

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Hi! My understanding of this optional-secrets was more simple. A secret that will be created by helm only if user supplies the values needed for this secret.

Is my view incorrect? Would this break the way helmify works with secrets?

Reply:
it is not incorrect but it is a diffrent approach. i also don't think that ir is easy to implement. for example, how to handle cases when secret has multiple values and not all of them are set?

in my personal opinion external secrets are a bit more user friendly for helm-chart user. what do you think?

Reply:
In my implementation I made it so that if the user defines any of the values of the secret, then all of them are required. I just defined a non rendering template when the secret map in values.yaml is not defined (i.e. the user does not want to create it). I believe this makes it simple to understand by users using the helm chart

Reply:
understood. i personally had use-cases when secret is already exists or created with a different tool. In this case it is very useful to set custom name for external secret which will be used in deployments and other manifests instead.

Reply:
Hi I have updated and removed the firstline issue